backend/routes/gameRoutes.js

```diff
@@ -6,9 +6,9 @@ const Game = require("../models/Game");
 router.post("/", async (req, res) => {
   const { name, platform, releaseYear, cover, category } = req.body;
 
+  try {
   if (!name) {
-    res.status(402).json({ error: "O nome é obrigatório" });
-    return;
+    return res.status(402).json({ error: "O nome é obrigatório" });
```

Comment:
You have to put this way, because its will return res.status... if you put the return after this, its will bring a unexpect error